CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+
+For example:
+
+> Add link to code of conduct in README.md
+
+Doesn't tell me anything about why you're doing that
+
+> Add link to code of conduct in README.md because users don't always look in the CONTRIBUTING.md
+
+Tells me the problem that you have found, and the pull request shows me the action you have taken to solve it.
+
+### Is it of good quality
+
+-   There are no spelling mistakes
+-   It reads well
+-   For english language contributions: Has a good score on [Grammarly](https://gist.github.com/PurpleBooth/grammarly.com) or [Hemingway App](http://www.hemingwayapp.com/)
+-   For code contributions: It should minimally pass the automated code quality checks such as linting and tests, as well was having new tests of its own.
+
+### Does it move this repository closer to my vision for the repository
+
+The aim of this repository can be found in [README.md](README.md)
+
+### Does it follow the contributor covenant
+
+This repository has a [code of conduct](CODE_OF_CONDUCT.md), This repository has a code of conduct, I will remove things that do not respect it.
+
+
+## Getting Started
+
+### Ready to make a change? Fork the repo
+
+Fork using GitHub Desktop:
+
+- [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/getting-started-with-github-desktop) will guide you through setting up Desktop.
+- Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
+
+Fork using the command line:
+
+- [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.
+
+Fork with [GitHub Codespaces](https://github.com/features/codespaces) (if available for this repository):
+
+- [Fork, edit, and preview](https://docs.github.com/en/free-pro-team@latest/github/developing-online-with-codespaces/creating-a-codespace) using [GitHub Codespaces](https://github.com/features/codespaces) without having to install and run the project locally.
+
+### Make your update:
+Make your changes to the file(s) you'd like to update. 
+  - Are you making changes to the application code? See [README.md](README.md) for more information on how to run this project locally.
+  - Are you contributing to documentation in this repository? We use [GitHub Markdown](contributing/content-markup-reference.md).
+
+### Open a pull request
+When you're done making changes and you'd like to propose them for review, use the pull request button to open your PR (pull request).
+
+### Submit your PR & get it reviewed
+- Once you submit your PR, core contributors from this repository will review it with you. The first thing you're going to want to do is a [self review](#self-review).
```

Comment:
Should we link to team page or add names to contact in case ?

Reply:
in case of?

Reply:
problems to open PR / question / dunno

Reply:
nah if github is down i don’t want to be dealing with it
if they don’t know how to do a pr they can read this document again and
follow the links